tests/acceptance/test_stdlib.py

```diff
@@ -51,4 +51,16 @@ def test_libmodule(test_module_location, test_module_name):
             assert ex.code != 32
             return
 
-        assert False, "shouldn't get there"
+
+@pytest.mark.acceptance
+@pytest.mark.parametrize(
+    ("test_module_location", "test_module_name"), MODULES_TO_CHECK, ids=MODULES_NAMES
+)
+def test_lib_module_no_error(test_module_location, test_module_name):
+    os.chdir(test_module_location)
+    with _patch_stdout(io.StringIO()):
+        try:
+            pylint.lint.Run([test_module_name, "--errors-only", "--ignore=test"])
+        except SystemExit as ex:
+            assert ex.code == 0
```

Comment:
During testing I found that there are actually a number of `stdlib` that due return an error. For example `shutil`.